Sources/MapboxMaps/Gestures/GestureRecognizerDelegate.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+internal class GestureRecognizerDelegate: NSObject, UIGestureRecognizerDelegate {
+
+    /// The view that all gestures operate on
+    private weak var view: UIView?
```

Comment:
Let's remove this weak ref to view and just grab the view from the gesture recognizer in `gestureRecognizerShouldBegin`